recipe/meta.yaml

```diff
@@ -1,4 +1,8 @@
 {% set version = "5.1.2" %}
+# TODO: investigate upstream (whether it is expected these should exist)
+{% set k_skips = ["upgrade_v4_to_4_dot_5", "sample_notebook"] %}
```

Comment:
Do these need to be jinja vars? I'm not a big fan of those and they'll go away in the recipe format.